src/contract_wrappers/modules/sto/usd_tiered_sto_wrapper.ts

```diff
@@ -964,6 +971,7 @@ export default class USDTieredSTOWrapper extends STOWrapper {
     from: string,
     investmentValue: BigNumber,
     fundRaiseType: FundRaiseType,
+    usdToken: string | undefined,
```

Comment:
```suggestion
    usdToken?: string,
```
Make it optional, so you don't need to send `undefined`